connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -267,6 +268,20 @@ public String memberId() {
         return JoinGroupRequest.UNKNOWN_MEMBER_ID;
     }
 
+    @Override
+    protected void handlePollTimeoutExpiry() {
+        Stage currentStage = listener.onPollTimeoutExpiry();
+        log.warn("worker poll timeout has expired. This means the time between subsequent calls to poll() " +
+            "in DistributedHerder tick() method was longer than the configured rebalance.timeout.ms. " +
```

Comment:
From a user perspective, the class and method names are irrelevant, and bringing up irrelevant details in diagnostics can be misleading. "The last thing the worker was doing was: {} and may contribute to this timeout" is much more understandable and still gets the point across.

Reply:
Got it, yeah I changed the message accordingly. I tried to keep the logline similar to the one in consumer's poll timeout but i think it's fine to deviate in this case.